src/scrabble.js

```diff
@@ -1,5 +1,59 @@
-function scrabble() {
+const array1 = ['A', 'E', 'I', 'O', 'U', 'L', 'N', 'R', 'S', 'T']
+const array2 = ['D', 'G']
+const array3 = ['B', 'C', 'M', 'P']
+const array4 = ['F', 'H', 'V', 'W', 'Y']
+const array5 = ['K']
+const array8 = ['J', 'X']
+const array10 = ['Q', 'Z']
 
+function letterValue(letter) {
+  let value = 0
+  const letterUpper = letter.toUpperCase()
+  // console.log(letterUpper)
+  if (array1.includes(letterUpper)) {
+    value += 1
+    // console.log(value)
+    return value
+  } else if (array2.includes(letterUpper)) {
+    value += 2
+    return value
+  } else if (array3.includes(letterUpper)) {
+    value += 3
+    return value
+  } else if (array4.includes(letterUpper)) {
+    value += 4
+    return value
+  } else if (array5.includes(letterUpper)) {
+    value += 5
+    return value
+  } else if (array8.includes(letterUpper)) {
+    value += 8
+    return value
+  } else if (array10.includes(letterUpper)) {
+    value += 10
+    return value
+  } else {
+    return value
+  }
 }
 
+// function multiLetter(word) {
+//   if (word.includes('{')) {
+//   }
+// }
+
+function scrabble(word) {
+  let score = 0
+  if (typeof word !== 'string' || word instanceof String) {
+    return score
+  }
+  for (let i = 0; i < word.length; i++) {
+    score += letterValue(word[i])
+    // console.log(score)
+  }
+  return score
+}
+
+// console.log(scrabble('cabbage'))
+
 module.exports = scrabble
```

Comment:
I will combine my questions here:
1)  On line 47 - you use instanceof String. - What are you looking for with this command? I can see it's like your checker to make sure the information in the `word` is right, but what specifically does that command do?

2) What made you decide on arrays for your points data?

3)  What is your plan for implementing the letter and word multipliers into this code?